CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 0.2.3 04.12.2024
```

Comment:
nit: should that be 0.2.4 to match the version in `Cargo.toml`?

Reply:
Yep, https://github.com/alekseysidorov/include-utils/pull/10